plugins/user-authenticators/ldap/src/test/resources/ldapunit.ldif

```diff
@@ -0,0 +1,151 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Same as previous file, do we really need it, since we now have ldap_test_data.py?